src/GraphLayout.jl

```diff
@@ -2,11 +2,30 @@ __precompile__(true)
 
 module GraphLayout
     using Compose   # for plotting features
+    using GeometryTypes   # for generating the layout
 
     typealias AdjList{T} Vector{Vector{T}}
 
+    type Node
```

Comment:
These should be immutables, otherwise they will be heap allocated and arrays of them will be a list of pointers. (slooow!)